components/scripts/lib/build-scan-parse.sh

```diff
@@ -74,28 +73,48 @@ debug_build_scan_data() {
 
 # shellcheck disable=SC2034 # not all scripts use all of the fetched data
 parse_build_scan_row() {
-  local build_cache_metrics_only="$1"
-  local build_scan_row="$2"
+  local build_scan_row="$1"
 
   local run_num
 
   while IFS=, read -r run_num field_1 field_2 field_3 field_4 field_5 field_6 field_7 field_8 field_9 field_10 field_11 field_12 field_13 field_14 field_15 field_16 field_17 field_18 field_19 field_20 field_21; do
```

Comment:
💡The more we work with this code, the more it bugs me that these are all called "field_N". At some point, we really should try to rename them.